libs/@guardian/libs/src/measure/@types/measure.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Calling this method ends the measurement
+ * @returns {number} duration, rounded up to the nearest millisecond
+ */
+export type MeasureEnd = () => number;
+
+declare global {
+	interface Performance {
+		/**
+		 * Firefox returned `undefined` before v101
+		 * @see https://developer.mozilla.org/en-US/docs/Web/API/Performance/mark#browser_compatibility
+		 */
+		measure(
+			measureName: string,
+			startOrMeasureOptions?: string | PerformanceMeasureOptions,
+			endMark?: string,
+		): PerformanceMeasure | undefined;
+	}
+}
```

Comment:
Is there a better way to do this? The alternative is to disable [the `no-unnecessary-condition` ESLint rule](https://typescript-eslint.io/rules/no-unnecessary-condition/)